### ISSUE TITLE
Cannot boot new installation

### ISSUE THREAD

**Issue by cgamez** [clearlinux/distribution#766]
USB live-29460 works fine and proceed to HDD installation.
hard disk installation crashes when it displays following message:
fb0: switching to inteldrmfb from EFI VGA
Laptop Machine will power off immediately.
Same issue with img dd to HDD (img to virtualbox works fine)
CPU: Intel Celeron N3050 @ 1.60 GHz
CPU ID: 406C3
Laptop: DELL Inspiron 11-3162
BIOS: 2.4.0 (newest posted by DELL)
Tried adding nomodeset to kernel parameters, but could not find where to change parameters using live USB.  Fixing boot instructions said repaired 1 file but issue is the same.  Tried to add boot.  Also tried adding delay time using clr-boot-manager set-timeout but only works on running kernel not in HDD.  Thanks.

**Comment by jaycenet** [comment]
I have the same problem

**Comment by ahkok** [comment]
Are you booting using UEFI enabled?

**Comment by jaycenet** [comment]
Yes EFI enabled, and a entry is created in the BIOS 'Linux Boot'

But this message is appearing at boot : 

https://pasteboard.co/IfBP5hi.jpg

I checked on the /boot partition the files are present

**Comment by cgamez** [comment]
@ahkok Yes, EFI enabled, snapshot of boot messages (just before shutdown)
![Screenshot_20190518-232748](https://user-images.githubusercontent.com/31857373/58046265-6b347480-7b0a-11e9-8126-e0963aa600c5.png)

**Comment by miguelinux** [comment]
Hit down arrow key when booting to show "boot menu".
then hit "e" key to edit and add `nomodeset` to the kernel parameters

**Comment by cgamez** [comment]
@miguelinux I tried to add a 20 second menu delay but never found how to add it in the HDD while using the LIVE USB.  I will try the down arrow key and post results here.  The EFI loader is at BIOS level, I have tried all loader and EFI options (conf and kernel).  Do you know which file is the one that will show the boot menu?

**Comment by miguelinux** [comment]
> @miguelinux I tried to add a 20 second menu delay but never found how to add it in the HDD while using the LIVE USB.  I will try the down arrow key and post results here.  The EFI loader is at BIOS level, I have tried all loader and EFI options (conf and kernel).  Do you know which file is the one that will show the boot menu?

They are in the ESP (EFI System Partition), and usually it is mounted in
/boot

the safer way to do it is using "clr-boot-manger" 

sudo clr-boot-manager set-timeout 20
sudo clr-boot-manager update

the raw (not safer) way is to edit the files directly 

sudo systemctl start boot.mount
sudo nano /boot/loader/loader.conf

and add  the following line:

timeout 20

You can find more info at "man clr-boot-manager", also CL uses
systemd-boot, you can find more info in the web.

**Comment by cgamez** [comment]
@miguelinux thanks for all your information.  I had tried using clr-boot-manager but it did not work with the Live USB onto the HDD, only in virtualbox it worked fine.
I had installed other distro but reinstalled Clear OS and magically it works now!  I did not have to edit anything, but I appreciate your manual edit, yesterday I tried but could not find the correct file to edit.
The issue is solved but by unknown reasons.  Used same setup, same USB.
